guide/ops/locations/customizing_location_ports.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Customizing Location Ports
+layout: website-normal
+---
+
+## Opening ports during startup.
+
+Use the `org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer` location customizer.
```

Comment:
I think these docs need more context, and caveats.

For example, the `SharedLocationSecurityGroupCustomizer` only works in clouds that use the security group abstraction (so not in Azure, SoftLayer, etc).